AdminUi/apps/admin_ui/lib/home/identity_details/identity_details.dart

```diff
@@ -0,0 +1,239 @@
+import 'dart:io';
+
+import 'package:admin_api_sdk/admin_api_sdk.dart';
+import 'package:admin_api_types/admin_api_types.dart';
+import 'package:admin_ui/core/core.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:intl/intl.dart';
+
+class IdentityDetails extends StatefulWidget {
+  final String address;
+
+  const IdentityDetails({required this.address, super.key});
+
+  @override
+  State<IdentityDetails> createState() => _IdentityDetailsState();
+}
+
+class _IdentityDetailsState extends State<IdentityDetails> {
+  static const noTiersFoundMessage = 'No tiers found.';
+
+  Identity? _identityDetails;
+  List<TierOverview>? _tiers;
+  String? _selectedTier;
+
+  late final ScrollController _scrollController;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollController = ScrollController();
+    _reloadIdentity();
+    _reloadTiers();
+  }
+
+  @override
+  void dispose() {
+    _scrollController.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if (_identityDetails == null) return const Center(child: CircularProgressIndicator());
+
+    return Scrollbar(
+      controller: _scrollController,
+      child: SingleChildScrollView(
+        controller: _scrollController,
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: [
+            if (Platform.isMacOS || Platform.isWindows) const BackButton(),
+            _buildIdentityCard(context),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildIdentityCard(BuildContext context) {
```

Comment:
[don't return widgets from functions](https://www.youtube.com/watch?v=IOyq-eTRhvo) (!!!!!!!!!!)

Reply:
Changes made.

Reply:
actually, not really ... why didn't you create a helper widget for that?

Reply:
For the whole card?